src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -2619,7 +2619,8 @@ void AuraEffect::HandleAuraUntrackable(AuraApplication const* aurApp, uint8 mode
 
 void AuraEffect::HandleAuraModPetTalentsPoints(AuraApplication const* aurApp, uint8 mode, bool /*apply*/) const
```

Comment:
What causes this fatal error in Travis?
```
src/server/game/Spells/Auras/SpellAuraEffects.cpp:2620:71: fatal error: 
      unused parameter 'aurApp' [-Wunused-parameter]
void AuraEffect::HandleAuraModPetTalentsPoints(AuraApplication const* aurApp...
                                                                      ^
```
Is it caused by a change in this PR or something outside of it?